dspace/config/modules/authority.cfg

```diff
@@ -0,0 +1,15 @@
+#---------------------------------------------------------------#
+#----------------- AUTHORITY CONFIGURATIONS --------------------#
+#---------------------------------------------------------------#
+# These configs are used by the authority framework             #
+#---------------------------------------------------------------#
+
+#####  Authority Control Settings  #####
+
+plugin.named.org.dspace.content.authority.ChoiceAuthority = \
+org.dspace.content.authority.EPersonAuthority = EPersonAuthority
+
+
+choices.plugin.dspace.object.owner = EPersonAuthority
+choices.presentation.dspace.object.owner = suggest
+authority.controlled.dspace.object.owner = true
```

Comment:
These 3 settings have no description. They need a description so that others know how they are used & what values they may accept.  They are not self explanatory

Reply:
These three new settings still need a description and examples of possible values.  They are undocumented currently.

Reply:
@LucaGiamminonni : I think you misunderstand my point.  It appears these three settings are **required** for  Researcher Profiles to work properly.  If you notice, in every other example in this "authority.cfg", there's some sort of basic comment that says things like..

```
# Configuration settings for ORCID based authority control.
# Uncomment the lines below to enable configuration
```

But, in this code, you've given users no information about these settings.  I think they MUST have comments to describe their purpose. 

Please add something like:
```
# Configuration settings required for Researcher Profiles
# These settings ensure "dspace.object.owner" field are indexed by Authority Control
```
(I'm not sure if that text is correct, but that's essentially what I'm asking for.  These settings need to document how/why they are used. That's required of all configurations.)